puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: {timeout}
```

Comment:
Turns out, I was getting a tuple index error when the timeout parameter was indented 4 spaces and I interpreted that incorrectly. The real culprit was the `{0}` not being escaped after introducing the parameter. Updated the other jobs based on your suggestion.